### PR TITLE
Use roman numerals like on the picture described

### PR DESCRIPTION
**Pull request by DarthGandalf**
Also Heroes -> Hero

to be merged after 1.3, because this is not important enough to trouble translators

en-all

### CODE REVIEW

**Comment by qwrtln** [comment]
All the languages (maybe apart from Hebrew), use the same numerals, so maybe you can do it without introducing too much fuzziness?

**Comment by DarthGandalf** [comment]
I cannot update Heroes/Hero without knowing all the languages

**Comment by davidpodless** [comment]
> I've tried to update translations. PTAL
> 
> @davidpodless I tried to update the Setup section, but please check whether the result looks correct. I didn't dare updating other sections.
> 
> @DerSchlupf I left the description of the mine fuzzy, because the translation seemed incomplete to me and didn't mention the units at all

Fine by me, +1

BTW, Im doing a typo fixing now anyway, so Im fine with that being fixed now

**Review by davidpodless — APPROVED** [review]

**Comment by DerSchlupf** [comment]
> I've tried to update translations. PTAL
> 
> @davidpodless I tried to update the Setup section, but please check whether the result looks correct. I didn't dare updating other sections.
> 
> @DerSchlupf I left the description of the mine fuzzy, because the translation seemed incomplete to me and didn't mention the units at all

Yes I had to short the Text to get everything on one Page. :( And since the mine example is about the income of the mine, not about guarding it, I thought the part with the units was dispensable. Is it a problem?

**Comment by DerSchlupf** [comment]
Edit: I've review your changes and that the Part with the mine is now still English but on one Page. I will translate it a second time in the next days and perhaps there is enough space for the full translation. So its fine for me, too!

**Review by DerSchlupf — APPROVED** [review]

**Comment by DarthGandalf** [comment]
> still English but on one Page

Yeah, that's what fuzzy does. It turns off the translated paragraph, turning it back to English until the translation is update to take the changes to original into account (and the fuzzy marker removed)

> And since the mine example is about the income of the mine, not about guarding it, I thought the part with the units was dispensable

What's the proper translation which doesn't fit?

**Comment by DerSchlupf** [comment]
It's a while ago, that I dont know which Variante I've tested, but the following should be a hopefully short one:

"Diese Mine, bewacht von neutralen Einheiten der Stufe 3, produziert [K]. Der erste Spieler, der die Mine einnimmt, erhält sofort ein [K] und erhöht zusätzlich sein Einkommen um eine Stufe."

More shorter with following changes:
"... bewacht von neutralen Einheiten Stufe 3..."
"...Der erste Spieler, der  einnimmt..."
"...und erhöht sein Einkommen..."

But the word "zusätzlich" in the third suggestion would be nice, if it could stay. It's the translation of "in addition"

**Comment by DarthGandalf** [comment]
Works fine, with a small hack:

de-all

**Comment by davidpodless** [comment]
I've fixed the conflict we had with previous commit

**Review by qwrtln — APPROVED** [review]

File: sections/all_map_locations.tex [hunk 1]
```diff
@@ -156,7 +156,7 @@ \subsection*{Towns, Mines and Settlements}
 
 \begin{center}
   \includegraphics[width=0.55\linewidth]{\images/mine_example.png}\\
-  \textit{A mine that produces \svg{valuablegreater}, guarded by Level 3 Neutral Units.
+  \textit{A mine that produces \svg{valuablegreater}, guarded by Level III Neutral Units.
```
Comment (qwrtln) [inline sections/all_map_locations.tex:159]:
```suggestion
  \textit{A mine that produces \svg{valuablegreater}, guarded by \mbox{Level III} Neutral Units.
```
There should be no line break here.

**Comment by qwrtln** [comment]
@davidpodless Are you deliberately using "3" instead of "III" in All Map Locations now? I left out Hebrew as fuzzy because of that.

**Comment by DarthGandalf** [comment]
I disagree with the latest commit - I think the translations should have mbox too, because something else may change, moving this text and moving this linebreak.

**Comment by qwrtln** [comment]
OK, revert it.

**Comment by davidpodless** [comment]
> @davidpodless Are you deliberately using "3" instead of "III" in All Map Locations now? I left out Hebrew as fuzzy because of that.

No, I'll do it later with a new commit

I want to see how it looks in the pdf before commiting the change

**Comment by DarthGandalf** [comment]
> OK, revert it.

Done. I added mbox to Polish too. Skipped Ukrainian, because I don't know whether "III-го" looks correct.

**Comment by davidpodless** [comment]
@qwrtln you didn't merge this just yet, so I've added the unfuzzy here :-)

**Review by Heegu-sama — APPROVED** [review]
Good change for consistency.

**Pull request merged**